cmd/ui-errors.go

```diff
@@ -0,0 +1,170 @@
+MINIO_STORAGE_CLASS_RRS: Format "EC:<Default_Parity_Reduced_Redundancy_Class>" (e.g. "EC:3"). This sets the number of parity disks for Minio server in reduced redundancy mode. Objects are stored in Reduced Redundancy mode, if Put request specifies RRS storage class.
+Refer to the link https://github.com/minio/minio/tree/master/docs/erasure/storage-class for more information.`,
+	)
+
+	uiErrUnexpectedBackendVersion = newUIErrFn(
+		"Backend version seems to be too recent",
+		"Please update to the latest Minio version",
+		"",
+	)
+
+	uiErrInvalidAddressFlag = newUIErrFn(
+		"--address input is invalid",
+		"Please check --address parameter",
+		`--address binds a specific ADDRESS:PORT, ADDRESS can be an  IP or hostname (default:':9000')
+    Examples: --address ':443'
+              --address '172.16.34.31:9000'`,
+	)
+
+	uiErrInvalidFSEndpoint = newUIErrFn(
+		"The given endpoint is not suitable to activate standalone FS mode",
+		"Please check the given FS endpoint",
+		`FS mode requires only one writable disk path.
+Example 1:
+   $ minio server /data/minio/`,
+	)
+
+	uiErrUnableToWriteInBackend = newUIErrFn(
+		"Unable to write to the backend",
+		"Please ensure that Minio binary has write permissions to the backend",
+		"",
+	)
+
+	uiErrUnableToReadFromBackend = newUIErrFn(
+		"Unable to read from the backend",
+		"Please ensure that Minio binary has read permission from the backend",
+		"",
+	)
+
+	uiErrPortAlreadyInUse = newUIErrFn(
+		"Port is already in use",
+		"Please ensure no other program is using the same address/port",
+		"",
+	)
+
+	uiErrNoPermissionsToAccessDirFiles = newUIErrFn(
+		"Missing permissions to access to the specified path",
+		"Please ensure the specified path can be accessed",
+		"",
+	)
+
+	uiErrSSLUnexpectedError = newUIErrFn(
+		"Invalid TLS certificate",
+		"Please check the content of your certificate data",
+		`Only PEM (x.509) format is accepted as valid public & private certificates.`,
+	)
+
+	uiErrSSLUnexpectedData = newUIErrFn(
+		"Something is wrong with your TLS certificate",
+		"Please check your certificate",
+		"",
+	)
+
+	uiErrSSLNoPassword = newUIErrFn(
+		"no password was specified",
+		"Please set the password to this environment variable `"+TLSPrivateKeyPassword+"` so the private key can be decrypted",
+		"",
+	)
+
+	uiErrNoCertsAndHTTPSEndpoints = newUIErrFn(
+		"HTTPS is specified in endpoint URLs but no TLS certificate is found on the local machine",
+		"Please add a certificate or switch to HTTP.",
+		"Refer to https://docs.minio.io/docs/how-to-secure-access-to-minio-server-with-tls for information about how to load a TLS certificate in th server.",
+	)
+
+	uiErrCertsAndHTTPEndpoints = newUIErrFn(
+		"HTTP is specified in endpoint URLs but the server in the local machine is configured with a TLS certificate",
```

Comment:
`HTTP is specified in endpoint URLs but ...`
=>
`HTTP is specified in endpoint`~~`URLs`~~`but ...`